tests/webapp_t.py

```diff
@@ -278,7 +278,7 @@ class WebappIntegration(loris_t.LorisTest):
     def test_index(self):
         resp = self.client.get('/')
         self.assertEqual(resp.status_code, 200)
-        self.assertTrue(resp.data.startswith('This is Loris, '))
+        self.assertTrue(resp.data.decode('utf8').startswith('This is Loris, '))
```

Comment:
Suggestion: rather than decoding `resp.data`, it might be cleaner to do the comparison with a bytestring:

```python
self.assertTrue(resp.data.startswith(b'This is Loris, '))
```

Ditto all the tests below.